test/junit/scala/collection/Sizes.scala

```diff
@@ -5,15 +5,39 @@ import scala.collection.mutable.ListBuffer
 import scala.tools.testing.JOL
 import scala.tools.testing.JOL.assertTotalSize
 
+
+object Sizes {
+  def list: Int = 24
+  def listBuffer: Int = 32
+
+  def refArray(length:Int): Int = {
+    (16 + (length+1) * 4) /8 * 8
+  }
+  def wrappedRefArray(length:Int): Int = refArray(length) + 24
```

Comment:
```
java.lang.AssertionError:  footprint:
     COUNT       AVG       SUM   DESCRIPTION
         1        24        24   [Ljava.lang.String;
         1        16        16   scala.collection.mutable.WrappedArray$ofRef
         2                  40   (total)
 
Expected :48
Actual   :40
```

I don't know much about object allocations, but looking at https://stackoverflow.com/a/258150/248998, `WrappedArray$ofRef` has a 12 byte header and a single 4 byte pointer (as we're in a small heap with UseCompressedOops).

So should it be `+ 16` here?

Changing it to 16 fixed all of the `largeListAllocation`, `largeSeqAllocation` and `wrappedArray2` tests for me. Is it different in your environment?

Reply:
looks like it. 

Incidentally that why I added the trace (thanks @retronym for the pointer) so you can see from ` 1        16        16   scala.collection.mutable.WrappedArray$ofRef` that it should be 16